Tests/XcodeGenKitTests/FixtureTests.swift

```diff
@@ -17,13 +17,47 @@ func generate(specPath: Path, projectPath: Path) throws {
     if newProject != oldProject {
         throw failure("\(projectPath.string) has changed. If change is legitimate commit the change and run test again")
     }
+
+    return newProject
 }
 
 func fixtureTests() {
 
     describe("Test Project") {
+        var project: XcodeProj?
+
         $0.it("generates") {
-            try generate(specPath: fixturePath + "TestProject/spec.yml", projectPath: fixturePath + "TestProject/GeneratedProject.xcodeproj")
+            project = try generate(specPath: fixturePath + "TestProject/spec.yml", projectPath: fixturePath + "TestProject/GeneratedProject.xcodeproj")
+        }
+
+        $0.it("generates variant group") {
+            guard let project = project else { throw failure("Project is not generated") }
+
+            func getFileReferences(_ path: String) -> [PBXFileReference] {
+                return project.pbxproj.fileReferences.filter { $0.path == path }
+            }
+
+            func getVariableGroups(_ name: String?) -> [PBXVariantGroup] {
+                return project.pbxproj.variantGroups.filter { $0.name == name }
+            }
+
+            let resourceName = "LocalizedStoryboard.storyboard"
+            let baseResource = "Base.lproj/LocalizedStoryboard.storyboard"
+            let localizedResource = "en.lproj/LocalizedStoryboard.strings"
+
+            guard let variableGroup = getVariableGroups(resourceName).first else { throw failure("Couldn't find the variable group") }
+
+            do {
```

Comment:
Is this just for visual grouping? Otherwise the `it` function is throwing so you don't need to wrap in a do

Reply:
Yes it's just for visual grouping.